terraform-plans/templates/github/SECURITY.md.tftpl

```diff
@@ -0,0 +1,24 @@
+<!-- This file is centrally managed as a template file in https://github.com/canonical/solutions-engineering-automation -->
+<!-- To update the file: -->
+<!-- - Edit it in the canonical/solutions-engineering-automation repository. -->
+<!-- - Open a PR with the changes. -->
+<!-- - When the PR merges, the soleng-terraform bot will open a PR to the target repositories with the changes. -->
+
+# Security policy
+
+If the vulnerability affects a dependency of the project, a new version of the component including
+the updated dependency will be released in the respective store, meaning that no new feature will
+be included: the update will be built on top of the previously last released stable version.
+
+If the vulnerability affects our charm/snap code itself, a new version will be built including the
+security fix on top of the current main branch, meaning that the security update will potentially
+include new previously unreleased features.
```

Comment:
while this is our default overall statement, I find it misleading when applied to a specific security.md file in a project, especially one like dcgm. For example, it may seem to imply that we commit to rebuilding dcgm with updated dependencies when those dependencies carry bugfixes, while we are only mantaining the snap packaging and would therefore only bump the dcgm (or dcgm exporter) version as a whole. I suggest dropping these paragraphs from the default template. If we want to include more specific guidance we should modify this text appropriately for the individual projects.

Reply:
yes exactly, that initial generic description isn't precise enough to explain what we would do when we act as snap packagers. Which ftr I think should be:
* if a security fix is available in core we rebuild without bumping dcgm
* if a security fix is available in a new dcgm release we bump verisons
* if a security fix is available in a dependency of dcgm but no new dcgm release exists we wait

Reply:
@aieri if this template is generic and to be applied to all projects, where will the specific information that you mentioned live?